examples/source/components/amp-web-push.html

```diff
@@ -1,3 +1,10 @@
+<!---
+
+draft: true
```

Comment:
we should fix the sample. My suggestion would be to host the sample specific service worker in the same scope as the sample.

Reply:
Reason this is a draft is the same as for the others mentioned in the comment above. For this specific case: do you have suggestions on how we want to structure the logic for page specific routes? Creating one big `/documentation/examples/` router and handling everything there?